src/app/globals.css

```
@@ -223,6 +231,48 @@
   --shadow-2xl: var(--shadow-2xl);
 }
 
+@keyframes dashboard-unfold {
+  from {
```

Stylelint (17.14.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)